tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table_test.py

```diff
@@ -0,0 +1,194 @@
+                concept_id,
+                concept_code,
+                concept_name,
+                domain_id,
+                vocabulary_id,
+                concept_class_id,
+                valid_start_date,
+                valid_end_date)
+            VALUES
+              (1585266, 'PIIState_CA', 'PII State: CA', 'observation', 'PPI', 'answer', date('2017-04-24'), date('2099-12-31')),
+              (1585847, 'SexAtBirth_Female', 'Female', 'observation', 'PPI', 'answer', date('2017-05-22'), date('2099-12-31'))
+            """).render(project_id=self.project_id, dataset_id=self.dataset_id)
+
+        # test data for observation_ext table
+        observation_ext_data_query = self.jinja_env.from_string("""
+            INSERT INTO
+              `{{project_id}}.{{dataset_id}}.observation_ext` (
+                observation_id,
+                src_id
+              )
+            VALUES
+              (111, 'PPI/PM'),
+              (222, 'PPI/PM'),
+              (333, 'PPI/PM'),
+              (444, 'PPI/PM')
+                    """).render(project_id=self.project_id,
+                                dataset_id=self.dataset_id)
+
+        self.load_test_data([
+            person_data_query, observation_data_query, concept_data_query,
+            observation_ext_data_query
+        ])
+
+    def test_person_ext_creation(self):
+        """
+        Tests that the person_ext table is created as designed.
+
+        Validates pre conditions, tests execution, and post conditions based on the load
+        statements and the tables_and_counts variable.
+        """
+
+        tables_and_counts = [{
+            'fq_table_name':
+                '.'.join([self.fq_dataset_name, 'person_ext']),
+            'fields': [
+                'person_id', 'src_id', 'state_of_residence_concept_id',
+                'state_of_residence_source_value', 'sex_at_birth_concept_id',
+                'sex_at_birth_source_concept_id', 'sex_at_birth_source_value'
+            ],
+            'loaded_ids': [],
+            'cleaned_values': [(123, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female'),
+                               (345, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female'),
+                               (678, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female'),
+                               (910, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female')]
+        }]
+
+        self.default_test(tables_and_counts)
+
+    def tearDown(self):
+        tables = ['observation', 'observation_ext', 'concept', 'person']
+        for table in tables:
+            self.client.delete_table('.'.join([self.fq_dataset_name, table]),
+                                     not_found_ok=True)
+
+        super().tearDown()
```

Comment:
This is repeating stuff done in `curation/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py`.  I suggested adding it before because you had not previously added those four tables to `cls.fq_table_names` in the `setUpClass` method.

Reply:
ah ok that makes sense, removing now!